src/index.js

```diff
@@ -145,7 +145,13 @@ export default function commonjs ( options = {} ) {
 					}
 
 					if ( node.type === 'Identifier' ) {
-						if ( ( node.name in uses && !uses[ node.name ] ) && isReference( node, parent ) && !scope.contains( node.name ) ) uses[ node.name ] = true;
```

Comment:
The 'in'  here is why we don't put require in use, seemed easier then making this clause more complex